packages/sdk/src/actions/getQuote.ts

```diff
@@ -158,6 +158,7 @@ export async function getQuote(params: GetQuoteParams): Promise<Quote> {
     message,
     logger,
     apiUrl,
+    allowUnmatchedDecimals: "true",
```

Comment:
NIT: Internally we should be able to just use a true boolean right? Not a stringified version

Reply:
Ah yeah you're right!

Reply:
https://github.com/across-protocol/toolkit/pull/222/commits/713c2ccd1b38ea442bc1b8b2372e5ceab2c8a520